consensus/parlia/parlia.go

```diff
@@ -651,11 +651,11 @@ func (p *Parlia) verifyHeader(chain consensus.ChainHeaderReader, header *types.H
 		if header.RequestsHash != nil {
 			return fmt.Errorf("invalid RequestsHash, have %#x, expected nil", header.ParentBeaconRoot)
 		}
-	} else {
-		// TODO(Nathan): need a BEP to define this and `Requests` in struct Body
-		if !header.EmptyRequestsHash() {
-			return errors.New("header has wrong RequestsHash")
-		}
+		// } else {
+		// 	// TODO(Nathan): need a BEP to define this and `Requests` in struct Body
+		// 	if !header.EmptyRequestsHash() {
```

Comment:
why remove the code?

Reply:
noted, it is a temporarily workaround, will re-introduce the logic in another PR of BEP-440.